packages/wonder-stuff-ci/src/types.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * The information about a release branch.
+ */
+type MobileReleaseBranchInfo = {
```

Comment:
**suggestion:** Don't forget to export this type from `index.ts` so consumers of the API can use it.

`export type {MobileReleaseBranchInfo} from "./types";`

or

`export * from "./types";`